projects/noloan2/app/src/main/java/hasadna/noloan2/FirestoreClient.java

```diff
@@ -0,0 +1,53 @@
+package hasadna.noloan2;
+
+import android.util.Base64;
+
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.firestore.DocumentReference;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.MessageLite;
+
+import hasadna.noloan2.protobuf.SMSProto.SmsMessage;
+
+public class FirestoreClient {
+  private final String SPAM_DOCUMENT_PATH = "noloan/smss";
+  
+  // < - 22 > - 31 for removing and changing with the user name
+  private final String USER_SUGGEST_COLLECTION = "noloan/user_data/user/<username>/spam_suggestions";
+  
+  private FirebaseFirestore client;
+  
+  public FirestoreClient() {
+    client = FirebaseFirestore.getInstance();
+  }
+  
+  // Write the message to the Firestore
```

Comment:
Comments should only add new information.
In this case, `FirestoreClient.writeMessage()` has that information pretty much.